ocs_ci/utility/utils.py

```diff
@@ -1430,17 +1430,22 @@ def wait_for_co(operator):
 
 def censor_values(data_to_censor):
     """
-    This function censor values in dictionary keys that match pattern defined
-    in config_keys_patterns_to_censor in constants.
+    This function recursively censor string and numeric values in dictionary
+    keys that match pattern defined in config_keys_patterns_to_censor in
+    constants.
 
     Args:
         data_to_censor (dict): Data to censor.
 
     """
     for key in data_to_censor:
-        for pattern in constants.config_keys_patterns_to_censor:
-            if pattern in key:
-                data_to_censor[key] = '*' * 5
+        if isinstance(data_to_censor[key], dict):
+            censor_values(data_to_censor[key])
+        elif isinstance(data_to_censor[key], (str, int, float)):
+            for pattern in constants.config_keys_patterns_to_censor:
+                if pattern in key.lower():
+                    data_to_censor[key] = '*' * 5
+    return data_to_censor
```

Comment:
missing in doc string, probably from old PR, but you can fix it :)

Reply:
Doc string updated.